frontend/src/components/LoadData.tsx

```diff
@@ -515,12 +534,14 @@ export function LoadData(props: { children?: ReactNode }) {
     }
     startWatch()
 
-    const timeout = setInterval(processEventQueue, THROTTLE_EVENTS_DELAY)
+    const timeout = setInterval(processEventQueue, 500)
     return () => {
       clearInterval(timeout)
       if (evtSource) evtSource.close()
     }
-  }, [caches, mappers, setters, setSettings])
+    // this effect must only run once--it sets up the call to /events on the backend
+    // this should be [], but linter would complain
```

Comment:
I think it would be better to use the directive to silence the linter here, and to keep the comment explaining why you've done so. It looks to me like the list of dependencies is all static, so you could also point out in the comment that they shouldn't change.

Reply:
true that-- this confused me at first--I added some states to list and suddenly /events was being called multiple times, which is ng